ios-swift/CountlyTestApp-iOS-swift/AppDelegate.swift

```diff
@@ -18,11 +18,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate
         config.host = "https://YOUR_COUNTLY_SERVER"
         config.enableDebug = true
 
-//      config.features = [CLYPushNotifications, CLYCrashReporting, CLYAutoViewTracking] //Optional features
+//      config.features = [CLYFeature.pushNotifications, CLYCrashReporting, CLYAutoViewTracking] //Optional features
```

Comment:
Please also update "CLYCrashReporting" and "CLYAutoViewTracking"  with "CLYFeature."

Reply:
Done